src/backend/cuda/CMakeLists.txt

```diff
@@ -6,6 +6,8 @@ FIND_PACKAGE(Boost REQUIRED)
 INCLUDE("${CMAKE_MODULE_PATH}/CLKernelToH.cmake")
 INCLUDE("${CMAKE_MODULE_PATH}/FindNVVM.cmake")
 
+OPTION(USE_LIBDEVICE "Use libdevice for CUDA JIT" ON)
```

Comment:
I don't think `USE_LIBDEVICE` says anything about the variable without using the description. I would suggest either `BUILD_CUDA_WITH_LIBDEVICE` or `USE_CUDA_LIBDEVICE` or similar. Something that suggests it is associated with JIT in CUDA.